roadmap:
  - city: hengyang
    utm_zone: 49
    mappath: hengyang/V3.0.2
    version: V3.0.2

  - city: beijing
    utm_zone: 50
    mappath: beijing/V3.0
    version: V3.0

  - city: dali
    utm_zone: 47
    mappath: dali/V1.3
    version: V1.3
  
  - city: yantai
    utm_zone: 51
    mappath: yantai/V1.7
    version: V1.7

radarFusion:
  roadmap_shrink: 1
  is_use_roadmap: true
  is_show_roadmap: false
  single_radar_max_id: 1000
  #m/s
  lowspeed_threshold: 2.5
  lowspeed_speed_threshold: 1.5
  highspeed_speed_threshold: 1.5
  #m
  match_min_distance_thresh: 1.3
  lowspeed_distance_threshold: 2.0
  highspeed_distance_threshold: 3.0
  #size
  size_lowspeed_threshold: 6.0
  lowspeed_size_length_threshold: 1.5
  lowspeed_size_wideth_threshold: 1.0
  lowspeed_size_height_threshold: 1.8
  highspeed_size_length_threshold: 3.5
  highspeed_size_wideth_threshold: 2.0
  highspeed_size_height_threshold: 1.8


